packages/crypto/test/pbkdf2.node.test.ts

```diff
@@ -1,9 +1,31 @@
-import { bufferFromString, pbkdf2 } from '..';
+import { bufferFromString, pbkdf2 as pbkdf2Node } from '..';
+
+interface PBKDF2Node {
+  (
+    password: Uint8Array,
+    salt: Uint8Array,
+    iterations: number,
+    keylen: number,
+    algo: string
+  ): string;
+  register(
+    fn: (
+      password: Uint8Array,
+      salt: Uint8Array,
+      iterations: number,
+      keylen: number,
+      algo: string
+    ) => string
```

Comment:
should this return void? 
```suggestion
    ) => void
```

Reply:
Should this be [`BytesLike`](https://github.com/FuelLabs/fuels-ts/blob/159fe33c93bc2b3a96148f28e20cdd3d58e41fe8/packages/crypto/src/node/pbkdf2.ts#L34-L41)?

Reply:
Also do you have any thoughts on [this](https://github.com/FuelLabs/fuels-ts/pull/3046/files#r1734921368)?

Reply:
Right, oversight on my part.